.github/workflows/repo-sync.yml

```diff
@@ -0,0 +1,153 @@
+name: Repo Sync
+
+# **What it does**: HashiCorp Docs has two repositories: hashicorp/web-unified-docs (public) and hashicorp/web-unified-docs-internal (private).
+# This GitHub Actions workflow keeps the `main` branch of those two repos in sync.
+# **Why we have it**: To keep the open-source repository up-to-date
+# while still having an internal repository for sensitive work.
+# **Who does it impact**: Open-source.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */2 * * *' # Runs every 2 hours
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  repo-sync:
+    if: github.repository == 'hashicorp/web-unified-docs-internal' || github.repository == 'hashicorp/web-unified-docs'
+    name: Repo Sync
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2 2024-10-28
+
+      - name: Sync repo to branch
+        uses: repo-sync/github-sync@3832fe8e2be32372e1b3970bbae8e7079edeec88 # v2.3.0 2023-07-13
+        with:
+          source_repo: https://${{ secrets.CI_GITHUB_TOKEN }}@github.com/github/${{ github.repository == 'hashicorp/web-unified-docs-internal' && 'docs' || 'docs-internal' }}.git
```

Comment:
@heatlikeheatwave I think this is going to return 

'...@github.com/github/docs' or '...@github.com/github/docs-internal'

Shouldn't it be:

${{ github.repository == 'hashicorp/web-unified-docs-internal' && 'hashicorp/web-unified-docs' || 'hashicorp/web-unified-docs-internal' }}?